drivers/CAN.h

```diff
@@ -111,6 +111,15 @@ class CAN {
      * @endcode
      */
     CAN(PinName rd, PinName td);
+
+    /** Initialize CAN interface and set the frequency
+      *
+      * @param rd the rd pin
+      * @param td the td pin
+      * @param f the bus frequency in hertz
+      */
+    CAN(PinName rd, PinName td, int f);
```

Comment:
Can you please name the parameter f the full name ? does not make sense to short it as it was some loop counter? Follow what is in the another API (baud(int baudrate), frequency(int hz), etc).